HoneyBadgers.ConsoleApp/Repositories/MovieRepository.cs

```diff
@@ -84,5 +85,121 @@ private string StringValidation(int minLength, int maxLength)
 
             return input;
         }
+        public void PrintMovies()
+        {
+            Console.WriteLine("LIST OF MOVIES:");
+            foreach (var movie in Movies)
+            {
+                Console.ForegroundColor = ConsoleColor.Green;
+                Console.WriteLine(
+                    $"TITLE:{movie.Title}\nRELEASE YEAR: {movie.Year}\nDIRECTOR: {movie.Director}\nWRITER: {movie.Writer}\nACTORS: {movie.Actors}\nPLOT: {movie.Plot}\nGENRE: {movie.Genre}\nCOUNTRY: {movie.Country}\nRAITING: {movie.ImdbRating}");
+                Console.WriteLine("");
+                Console.WriteLine("");
+                Console.ResetColor();
+            }
+        }
+        public void InputChangesInMovieData()
+        {
+            Console.WriteLine("Enter title of the movie you wish to edit");
+            var titleInput = Console.ReadLine();
+            var selectedMovie = Movies.FirstOrDefault(mov => mov.Title.Equals(titleInput, StringComparison.OrdinalIgnoreCase));
+
+            if (selectedMovie == null) return;
+
+
+            Console.WriteLine($"Current movie title is: {selectedMovie.Title}. Type new title or press enter to continue without changes");
+            var title = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(title.Trim()))
```

Comment:
ten if się powtarza i dobrze by było zrobić to w pętli aby kiedy użytkownik poda nie poprawą wartość, na przykład dwie spacje, będzie mógł wprowadzić wartość jeszcze raz